CHANGELOG.md

```diff
@@ -6,6 +6,13 @@ This project adheres to [Semantic Versioning](http://semver.org/).
 ------------
 
 
+v1.2.3
```

Comment:
@simpixelated I didn't provide a link here, since it doesn't exist yet. Not sure what the process is here. Do you update once released? Or should I put the 1.2.3 link in anyway?